nss-cohort-36/bangazon-client-imagination-station-react
# Lr application views

# Description
All navbar links should change the URLS
Fixes # (issue)
## Type of change

# Testing Instructions
1. `git fetch --all`
2. `git checkout lr-application-views`
3. Click on all links in navbar, and they should change the url in the browser. Also, check application views to make sure all components need are there. 
I added functionality to make the navbar load only register and login links when logged out and there are if statement to redirect to login if the user is not logged in. 
The navbar should also be mobile responsive. 
The product creation form has material UI inputs implemented in it now. 
Login will alert you if you put the wrong username or password in.
# Checklist:
